Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -14937,8 +14964,7 @@ def main():  # pragma: no cover
             configured_max_fetch = arg_to_number(params['max_fetch'])
             queries = log_types_queries_to_dict(params)
             fetch_max_attempts = arg_to_number(params['fetch_job_polling_max_num_attempts'])
-            max_fetch = cast(MaxFetch, dict.fromkeys(queries, configured_max_fetch)
-                             | last_run.get('max_fetch_dict', {}))  # type: ignore[arg-type, operator]
+            max_fetch = cast(MaxFetch, dict.fromkeys(queries, configured_max_fetch))
```

Comment:
where is the max_fetch used now? why is this change needed?

Reply:
Talked about